sync3/handler/connstate.go

```diff
@@ -105,6 +109,9 @@ func (s *ConnState) load(ctx context.Context) error {
 		rooms = append(rooms, sync3.RoomConnMetadata{
 			RoomMetadata: *metadata,
 			UserRoomData: urd,
+			// Best-effort only. Not going to be accurate if you've sent
+			// bump_event_types to exclude membership events.
```

Comment:
Why?

Reply:
This is a brand-new invite for a room we don't know about. But I guess in this situation there is literally no better timestamp available to use for sorting.

Hmm. Suppose that
- you exclude membership events from bump event types, but are interested in message events
- the last two events in the room are a message event (ts 50) and the invite (ts 100).

The room will originally appear in the list at ts 100 based on the invite data. After joining, the poller will get the room timeline and set the LastInterestedEventTimestamp to 50, pushing the room down the timeline. That sounds like it'd surprise me as a user.

Reply:
Will leave a comment noting this and move on.